ci/code_checks.sh

```diff
@@ -85,27 +85,17 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
     MSG='Partially validate docstrings (RT02)' ; echo $MSG
     $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=RT02 --ignore_functions \
-    	    pandas.Series.align \
-	    pandas.Series.dt.total_seconds \
-	    pandas.Series.cat.rename_categories \
-	    pandas.Series.cat.reorder_categories \
-	    pandas.Series.cat.add_categories \
-	    pandas.Series.cat.remove_categories \
-	    pandas.Series.cat.remove_unused_categories \
 	    pandas.Index.all \
 	    pandas.Index.any \
 	    pandas.MultiIndex.drop \
 	    pandas.DatetimeIndex.to_pydatetime \
 	    pandas.TimedeltaIndex.to_pytimedelta \
-	    pandas.core.groupby.SeriesGroupBy.apply \
-	    pandas.core.groupby.DataFrameGroupBy.apply \
 	    pandas.io.formats.style.Styler.export \
 	    pandas.api.extensions.ExtensionArray.astype \
 	    pandas.api.extensions.ExtensionArray.dropna \
 	    pandas.api.extensions.ExtensionArray.isna \
 	    pandas.api.extensions.ExtensionArray.repeat \
 	    pandas.api.extensions.ExtensionArray.unique \
```

Comment:
```suggestion
	    pandas.api.extensions.ExtensionArray.unique
```
I think you need to remove the backslash here, since this is the last in the list now